ArtOfIllusion/src/artofillusion/LayoutWindow.java

```diff
@@ -1325,6 +1326,7 @@ public ToolPalette getToolPalette()
 
   private void setDockableWidgetVisible(DockableWidget widget, boolean visible)
   {
+    System.out.println(visible);
```

Comment:
I think this got left in accidentally.

Reply:
Thanks. -- There's always one that get's away...